go/mysql/mysql56_gtid_set.go

```diff
@@ -334,6 +334,83 @@ func (set Mysql56GTIDSet) AddGTID(gtid GTID) GTIDSet {
 	return newSet
 }
 
+// Union implements GTIDSet.Union().
+func (set Mysql56GTIDSet) Union(other GTIDSet) GTIDSet {
+	if set == nil || other == nil {
+		return set
+	}
+	mydbOther, ok := other.(Mysql56GTIDSet)
+	if !ok {
+		return set
+	}
+
+	// Make a copy and add the new GTID in the proper place.
+	// This function is not supposed to modify the original set.
+	newSet := make(Mysql56GTIDSet)
+
+	for OtherSID, otherIntervals := range mydbOther {
```

Comment:
Was the capitalization of `OtherSID` intentional? We generally reserve that for package-exported symbols.

Reply:
Totally unintentional - good catch.

Reply:
Fixed.